.husky/set-node.sh

```
@@ -270,30 +281,31 @@ function get_package_manager() {
-
-  # 버전 정렬 함수(curl로 받아온 버전 문자열을 최신순으로 정렬)
-  function sort_versions() {
-    awk '{
-      split($0,v,/[+-]|\./);  # 버전 문자열을 배열로 분리
-      # 정렬을 위한 키와 원본 버전을 함께 저장
-      printf("%012d%012d%012d %s\n", v[1], v[2], v[3], $0);
-    }' |
-    sort -r |  # 역순 정렬로 최신 버전이 먼저 오도록 함
-    head -n 1 |  # 가장 최신 버전 선택
-    awk '{print $2}'  # 원본 버전 문자열 출력
+    local v1="$1"
```

Shellcheck (0.10.0)

[warning] In POSIX sh, 'local' is undefined.

(SC3043)